.travis.yml

```diff
@@ -23,8 +31,9 @@ matrix:
     - php: 5.4
       env: SOLR_VERSION="4.10.3" TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
 # 5.5
-    - php: 5.5
-      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="~2.8@dev"
+# Disabled, to be enabled when we want to support it around it's release.
+#    - php: 5.5
+#      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="~2.8@dev"
```

Comment:
Maybe replace it with regular Symfony version. Otherwise we won't do tests with PHP 5.5